app/views/_feedback_on_appeals_to_crown_court.html.haml

```diff
@@ -0,0 +1,10 @@
+.govuk-notification-banner{ role: "region", aria: { labelledby: "govuk-notification-banner-title" }, data: { module: "govuk-notification-banner" } }
```

Comment:
You could save some hassle if you use `= govuk_notification_banner(...)` here instead of using the HTML by hand - we added the govuk-components gem recently

Reply:
Actually, that's not easy to reuse, since in our case, the heading has a link inside it.
The example in:  https://govuk-components.netlify.app/components/notification-banner/#output-rendered-notification-banner-with-heading is similar, but not flexible enough (the link can be placed only at the end)